introduction.tex

```diff
@@ -1,53 +1,22 @@
 
-Over the last 5 years there has been an explosion of innovation through new cryptocurrencies. Unfortunately, it is very hard for a new cryptocurrency to gain traction. Without traction, and the market depth it ensues, a cryptocurrency will have difficulty getting listed in any exchange. But without being listed in any exchange a cryptocurrency cannot gain sufficient traction. The cryptocurrency can only be sold over-the-counter (OTC) by its miners or stakeholders, but the market for OTC trading and especially for an unknown cryptocurrency is traditionally very limited.
-
-This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, in order for it to gain traction. We recommend utilizing proof-of-burn in order to can allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy proof-of-work cryptocurrency.
-
-We wish to allow users to exchange a proof-of-work based cryptocurrency for a new cryptocurrency. We call the former the \emph{source cryptocurrency}, and the blockchain this cryptocurrency is based on the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
+While its adoption is undeniable, there has not been a formal treatment for proof-of-burn. This is the gap this work aims to fill.
 
 \noindent
 \textbf{Workflow.}
-A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things his receiving address on the target blockchain. The user then sends an amount of the source cryptocurrency to that address, essentially burning this amount in the source blockchain. He then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on his receiving address. The equivalent amount can be determined based to the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
+A user who wishes to burn her coins generates an address which we call a \emph{burn address} that encodes some metadata (called the \emph{tag}). She can then proceed to send any amount of cryptocurrency to the burn address. After burning her cryptocurrency, she may prove to any interested party that she has actually irrevocably destroyed the cryptocurrency in question.
```

Comment:
Shorten:

* you don't need the () around "called the \emph{tag}"
* remove the "has actually"
* She can then proceed -> She then proceeds
* she may prove -> she proves
* Remove the "in question"